projects/batfish/src/main/java/org/batfish/config/Settings.java

```diff
@@ -1065,10 +1077,12 @@ private void initConfigDefaults() {
     setDefaultProperty(BfConsts.ARG_ANALYSIS_NAME, null);
     setDefaultProperty(ARG_ANONYMIZE, false);
     setDefaultProperty(BfConsts.ARG_ANSWER_JSON_PATH, null);
-    setDefaultProperty(BfConsts.ARG_BDP_DEBUG_ALL_ITERATIONS, false);
-    setDefaultProperty(BfConsts.ARG_BDP_DEBUG_ITERATIONS_DETAILED, false);
-    setDefaultProperty(BfConsts.ARG_BDP_DEBUG_MAX_RECORDED_ITERATIONS, 12);
-    setDefaultProperty(BfConsts.ARG_BDP_DEBUG_REPEAT_ITERATIONS, false);
+    setDefaultProperty(BfConsts.ARG_BDP_DETAIL, false);
+    setDefaultProperty(BfConsts.ARG_BDP_MAX_OSCILLATION_RECOVERY_ATTEMPTS, 1);
```

Comment:
I would suggest making this default to `0` until we have more experience with recovery. Esp as we think we've fixed existing oscillation bugs in other ways.